tests/e2e/ctl_v3_move_leader_test.go

```diff
@@ -118,3 +118,14 @@ func testCtlV3MoveLeader(t *testing.T, cfg e2e.EtcdProcessClusterConfig) {
 		}
 	}
 }
+
+func setupEtcdctlTest(t *testing.T, cfg *e2e.EtcdProcessClusterConfig, quorum bool) *e2e.EtcdProcessCluster {
```

Comment:
Is it used solely in this file, or across many ?

If the latter - maybe we should move it to 'common_test.go' file ?

Reply:
Only `ctl_v3_move_leader_test.go` uses `setupEtcdctlTest`